tests/phpunit/includes/abstract-testcase.php

```diff
@@ -401,7 +401,7 @@ public static function flush_cache() {
 			)
 		);
 
-		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
```

Comment:
See above.

Reply:
It only removed `theme_json'` from each of these. https://core.trac.wordpress.org/changeset/55092